Testscripts/Linux/utils.sh

```diff
@@ -2056,7 +2056,7 @@ function detect_linux_distribution() {
 		fi
 	elif [ "$linux_distribution" == "ol" ]; then
 		linux_distribution='oracle'
-	elif echo "$linux_distribution" | grep -qi "debian"; then
+	elif echo "$linux_distribution" | grep -qi "debian" || [ "$linux_distribution" == "cbld" ]; then
```

Comment:
if echo "$linux_distribution" | grep -Eqi "debian|cbld"; then
is enough?

Reply:
Just out of curiousity what does "cbld" stands for? Is it a same debian distribution? If not don't we need to handle this condition in an elif block and set the linux distribution name to cbld?

Reply:
Common Base Linux-Delridge is linux distribution based on Debian and built from source by Microssoft: https://microsoft.sharepoint.com/teams/AzLinux54/SitePages/CBL-D.aspx?originalPath=aHR0cHM6Ly9taWNyb3NvZnQuc2hhcmVwb2ludC5jb20vOnU6L3QvQXpMaW51eDU0L0VldnQ2YXZpdE9kSmlELUU5b21SNXZRQnBraEItOG1RZm9vWDB4ZTUzM3NWYWc_cnRpbWU9QVUzSXBPc3MyVWc